openwisp_monitoring/device/apps.py

```diff
@@ -120,6 +121,11 @@ def connect_device_signals(self):
             sender=DeviceLocation,
             dispatch_uid='post_delete_devicelocation_invalidate_devicedata_cache',
         )
+        post_save.connect(
+            self.organization_post_save_receiver,
+            sender=Organization,
+            dispatch_uid='post_save_organization_clear_management_ip_monitoring_status',
```

Comment:
Maybe this can be shortened a bit?

Eg: `post_save_organization_disabled_monitoring`.